### DISCUSSION TITLE
Is there a way to hide a repo from contributing to the top languages stats?

### DISCUSSION THREAD

**Discussion by Mikael-Kolehmainen** [anuraghazra/github-readme-stats#1738] [Q&A] [answered]
Hey,

I have a Unity repo which essentially tops the top language stats at 86% and hence the stats don't give an accurate representations of my repos. Is there any way I could ignore the Unity repo from the stats, of course I could make it private and have all my other repos public but I don't want to do that. Is there for example a way to write in the hyperlink of the stats which repos to ignore, if there isn't, that would be a great future addition to github-readme-stats.

Mikael

**Comment by rickstaa** [comment, marked as answer]
Hey, @Mikael-Kolehmainen, thanks for your question. Using the ' exclude_repo ' option, you can exclude repos for the language card using the `exclude_repo ` option. For more information, see https://github.com/anuraghazra/github-readme-stats#language-card-exclusive-options.
